### ISSUE TITLE
Operator doesn't respond after some time

### ISSUE THREAD

**Issue by gilbeckers** [nolar/kopf#614]
## Long story short
Right after the deployment of my controller, everything works fine: he reacts on create/delete/update events of resources. Then, after leaving the controller for some time (<10min), it stops handling changes of resources and nothing is logged anymore (also when running with `--verbose` ) 

## Description

While the controller seems to be frozen, my liveness probe continues to be working. Also, if I restart the controller, all changes that where made in the meantime, are handled. 
I tested it on kopf 0.27, 0.28 and 0.28.2, all version give the same kind of behavior. 

I must be doing something wrong, but I don't have any idea what can be wrong. I don't think that operator is still stuck in a handler, because the last lines of the logs says something like (which means that the handler is finished, right?)
```
[2020-12-14 16:34:02,413] kopf.objects [INFO    ] [sample-project] Handler 'update_members/spec.members' succeeded.
[2020-12-14 16:34:02,414] kopf.objects [INFO    ] [sample-project] Update event is processed: 1 succeeded; 0 failed.
```
Has someone seen this behavior before? Or any ideas on how I can debug this one? 

## Environment

* Kopf version: 0.27, 0.28 and 0.28.2
* Kubernetes version: 1.16
* Python version: 3.7
* OS/platform: Ubuntu

**Comment by nolar** [comment]
@gilbeckers Can you please check #585 — does the solution from there works for you?

**Comment by gilbeckers** [comment]
@nolar It seems that adding the client and server timeouts fixed my issue! Thanks for your help and many thanks for the amazing framework :+1:

**Comment by nolar** [comment]
The issue should be fixed without any configuration in [1.44.0](https://github.com/nolar/kopf/releases/tag/1.44.0)

* #1267
